info/flow_and_pressure.md

```diff
@@ -0,0 +1,42 @@
+
+### Methods of Flow restriction
+
+#### Gicleur
+For most people, the maximum flow rate of a rotary pump is unnecessary and arguably detrimental.  Because they wet the puck so quickly, the pressure ramp is extremely fast and often causes inconsistencies due to the formation of channels. Thus, many cafes opt to restrict their maximum flow using a gicleur, which inserts before the grouphead in the hydraulic circuit and will slow down the pressure ramp for greater consistency. For normal espresso brewing, this will not change your maximum pressure.
+
+#### Needle-valve
+The needle valve was first pioneered by Slayer espresso, and it works like an adjustable gicleur, which in the Slayer implementation, allows you choose between a pre-brew setting at 2mL/s and a brew setting at 8mL/s water debit. On modern high-end e61 espresso machines, you can now opt to install flow control on those machines via a fully dynamic needle valve which will allow you to use a flow rate anywhere in between fully open and fully closed. This level of control will allow an e61 user to execute any flow profile, given that they have the skill to do so. 
+**Sidenote**: users who flow-profile on manual needle valve machines will often communicate by noting the water debit at a certain needle position, and indicating that position during the brew for a repeatable flow profile, while users who own a Decent DE1 will often communicate flow rate via the calculated flow before the grouphead. In a needle valve machine, the user is not "aiming" to get 4mL/s output flow when the needle is moved there. As we know, the output flow is reliant on the grind setting and will resist the input flow accordingly. Needle valve water debit then, is a reliable way to communicate flow without needing an digitized bluetooth scale.
```

Comment:
> the output flow is reliant on the grind setting and will resist the input flow accordingly.

This is confusing. Output flow should be equal to input flow + TDS. I think the TDS bit is probably not super significant here. A better way to explain this is to say that resistance will cause pressure buildup and therefore a flow rate that is lower than the water debit at that needle valve position.

> a~~n~~ digitized bluetooth scale.